frappe/utils/safe_exec.py

```diff
@@ -28,6 +30,18 @@ def default_function(*args, **kwargs):
 			return default_function
 		return ret
 
+def get_safe_query_builder():
+	"""Allows execution of SELECT SQL queries only.
+	Raises:
+	   	PermissionsError raised on execution of any other SQL query
+	"""
+	query_class = get_attr(str(frappe.qb).split("'")[1])
```

Comment:
I think you should initialize a new query builder instance instead. The CI failures seem to be related.